### PR TITLE
Enhance/improved up messaging

### PR DESCRIPTION
**Pull request by otakumike**
Introduces new `up` messaging and some various improvements for the way Tokaido spins up new environments - such as downloading fresh images and more comprehensive self-health checks

### CODE REVIEW

File: services/docker/containers.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	if len(containers) > 1 {
+		return "", fmt.Errorf("error looking up container state for container %s. Received %d matching containers when only wanting 1", name, len(containers))
+	}
+
+	if len(containers) < 1 {
+		utils.DebugErrOutput(fmt.Errorf("error looking up container state for container %s. Could not find a container by that name", name))
+		return "", nil
+	}
+
+	return containers[0].State, nil
```
Comment (yjimk) [inline services/docker/containers.go]: Should there be a check for `.State`? As in, if it's a `""` (falsy value for strings) should it fall back to reliable default?

File: services/tok/init.go [hunk 2]
```diff
@@ -70,18 +79,50 @@ func Init(yes, statuscheck bool) {
...
+		fmt.Println(Red("😓  Tokaido containers are not working properly"))
+
+		if !docker.StatusCheck("fpm") {
+			fmt.Println(Red("    The Tokaido FPM container failed to start up."))
+			fmt.Println(Red("    This most likely suggests a problem with your Drupal site that is causing PHP to crash."))
+			fmt.Println()
+			fmt.Println(Sprintf(Cyan("    You can try running '%s' to see the full PHP startup log"), Blue("tok logs fpm")))
+		} else if !docker.StatusCheck("nginx") {
+			fmt.Println(Red("    The Tokaido NGINX container failed to start up."))
+			fmt.Println(Red("    This is most likely caused to a Tokaido misconfiguration."))
```
Comment (yjimk) [inline services/tok/init.go]: typo

File: services/docker/containers.go (hunk 1 above)
Comment (otakumike) [reply services/docker/containers.go]: I don't think it ever would return an empty string, but good point. I've added this in now

File: services/tok/init.go (hunk 2 above)
Comment (otakumike) [reply services/tok/init.go]: 🙈

**Pull request merged**